src/cmd/commit.ts

```diff
@@ -32,52 +42,82 @@ export const gitPush = async (branch: string): Promise<boolean> => {
     d.info(e.stdout)
     d.error(e.stderr)
     return false
-  } finally {
-    cd(rootDir)
   }
 }
 
-export const commit = async (): Promise<void> => {
-  const d = terminal('commit')
-  await validateValues()
-  d.info('Preparing values')
-  const values = await hfValues()
-  cd(env.ENV_DIR)
-  await preCommit()
-  await encrypt()
-  d.info('Committing values')
-  cd(env.ENV_DIR)
+const setDeplymentStatus = async (): Promise<void> => {
+  const status = await getOtomiDeploymentStatus()
+  if (status !== 'deployed') {
+    await nothrow($`kubectl -n ${otomiStatusNamespace} create cm ${otomiStatusCmName} --from-literal=status='deployed'`)
+    // Since status is an indicator of successful deployment, the generated passwords must be deleted later.
+    await nothrow($`kubectl delete secret ${otomiPasswordsSecretName}`)
+  }
+}
+
+const setEnv = (values: any): void => {
+  const isCertStaging = values.charts?.['cert-manager']?.stage === 'staging'
+  if (isCertStaging) {
+    process.env.GIT_SSL_NO_VERIFY = 'true'
+    process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
+  }
+}
+const waitForGitea = async (values: any): Promise<void> => {
+  const giteaEnabled = values?.charts?.gitea?.enabled ?? true
+  if (giteaEnabled && (!env.CI || isChart)) {
```

Comment:
move this logic to where it belongs please...no more business awareness in dumb functions please please please

Reply:
make sense